### PR TITLE
Pre-fill registration form email w/ invite email

### PR DESCRIPTION
**Pull request by benvinegar**
Small quality-of-life improvement for the user invitation flow – prefills the registration form with the email used in the invitation.

/cc @getsentry/team

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/4032)

### CODE REVIEW

File: src/sentry/web/frontend/accept_organization_invite.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def handle(self, request, member_id, token):
             # Show login or register form
             request.session['_next'] = request.get_full_path()
             request.session['can_register'] = True
+            request.session['invite_email'] = om.email
```
Comment (dcramer) [inline src/sentry/web/frontend/accept_organization_invite.py]: it'd be good to pop this value out of the session at some point (ideally when can_register is cleared)

Comment (benvinegar) [inline src/sentry/web/frontend/accept_organization_invite.py]: I did on auth_login.py L86 – is there somewhere else this should be done?

Comment (dcramer) [inline src/sentry/web/frontend/accept_organization_invite.py]: jk somehow i missed that

**Comment by dcramer** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/4032)*

**Pull request merged**